butte-build/src/codegen.rs

```diff
@@ -461,7 +462,12 @@ impl ToTokens for ir::Table<'_> {
                 &quote!(butte::ForwardsUOffset),
                 true,
             );
-
+            let default_value = if let Some(default_value) = default_value {
+		let default_value = to_default_value(&ty_wrapped, &default_value);
```

Comment:
The indentation in this `let` looks off by 2.

Reply:
Though this particular block doesn't seem to correctly format with `cargo fmt`.

Reply:
My editor inserted two tabs instead of spaces; fixed.